xmtp_db/src/encrypted_store/association_state.rs

```diff
@@ -0,0 +1,194 @@
+use diesel::prelude::*;
```

Comment:
Any idea why this file isn't flagged as a `git mv`?

Reply:
not sure, it must've moved and then been added in a subsequent commit, i'll try to fix